andela/mmdp_site
# feat(responsive-map): Implement endpoint for number of services per lg

**What does this PR do?**
Implement an endpoint to the number of beneficiaries per local government in a state. It also returns the url of a state map on aws

**Description of Task to be completed?**
- Implement an endpoint to the number of beneficiaries per local government in a state.

**How should this be manually tested?**

* Check out  to this branch `ft-statemap-service-range-MMDP-60`
* Start development server.
* Make a get request to `http://0.0.0.0:3000/api/v1/state-map/Edo`

**Any background context you want to provide?**
- None

**What are the relevant Jira issues?**

[MMDP-60](https://viisaus.atlassian.net/browse/MMDP-60)

**Screenshots (if appropriate)**
<img width="1210" alt="Screenshot 2019-05-17 at 12 42 59 AM" src="https://user-images.githubusercontent.com/22221987/57931558-a50a3e80-78b0-11e9-8e3d-5c7d19df7030.png">


**Questions:**
- None
